### PR TITLE
Feature/unit members

### PR DESCRIPTION
**Pull request by jardrake**
~~This is **NOT DONE** just yet. We still need to discuss the stripNotes function in the old project.~~

We also need to touch up Units.Delete as it has several relationships that are deleted in the old project that aren't yet deleted here. It's probably outside the scope of this PR, but I'm not sure where to mention it.

### CODE REVIEW

File: src/API/Functions/UnitMembers.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		public static Task<IActionResult> UnitMembersGetOne(
+			[HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "memberships/{membershipId}")] HttpRequest req, int membershipId)
+			=> Security.Authenticate(req)
+				.Bind(_ => UnitMembersRepository.GetOne(membershipId))
+				.Bind(res => Pipeline.Success(res.ToUnitMemberResponse()))
+				.Finally(result => Response.Ok(req, result));
+
+		[FunctionName(nameof(UnitMembers.CreateUnitMembers))]
+		[OpenApiOperation(nameof(UnitMembers.CreateUnitMembers), UnitMembersTitle, Summary = "Create a unit membership", Description = "Authorization: Unit memberships can be created by any unit member that has either the `Owner` or `ManageMembers` permission on their unit membership. See also: [Units - List all unit members](#operation/UnitsGetAll).")]
+		[OpenApiRequestBody(MediaTypeNames.Application.Json, typeof(UnitMemberRequest), Required = true)]
```
Comment (jasonfrancis) [inline src/API/Functions/UnitMembers.cs:49]: The [generated documentation](http://localhost:8080/#operation/CreateUnitMembers) doesn't show `Role` as required, and shows the default value for `permissions` as "owner" when it's actually "viewer".

I'm not sure if it's worth tying ourselves in knots to get OpenApi honoring these data annotations, but I just wanted to point it out.

**Review by jasonfrancis — APPROVED** [review]
So far, so good.

File: src/API/Functions/UnitMembers.cs (hunk 1 above)
Comment (jardrake) [reply src/API/Functions/UnitMembers.cs:49]: As discussed, all the documentation presently is like this. Requireds only show for path parameters like relationshipId. The default values just seem to be ignored as well. I'm not sure, but I thought the `/// some description` where also supposed to map but don't seem to work either.

**Review by jhoerr — APPROVED** [review]
Awesomeness!

**Review by jasonfrancis — APPROVED** [review]

**Pull request merged**